caravel/assets/js_build.sh

```
@@ -1,4 +1,5 @@
 #!/bin/bash
+set -e
```

Why the set -e line? Doesn't that cause errors to fail silently?